docs/model_configuration.rst

```diff
@@ -158,6 +158,14 @@ minimal configuration above. Specifically:
   batching, the initial batch dimension must be variable-size for all inputs
   and outputs.
 
+* :ref:`PyTorch TorchScript <section-pytorch-models>` models have an optional
+  output configuration in the model configuration file to support cases where
+  there are variable number and/or datatypes of output. Both the input and
+  outputs in the configuration must follow a specific naming convention i.e.
+  "\<name\>__\<index\>". Where <index> is the index of the corresponding input or
```

Comment:
What is <name> in the naming convention? Can it be any string I want to use?

Reply:
<name> can be any string but must be followed by "__<index>" + where <index is the corresponding position of the I/O>

Reply:
You need to explain that better somewhere. Perhaps in model_configuration.rst at the top where it is talking about inputs and outputs. You could refer to that section from here.

Reply:
Ok. Will do